tests/suites/test_suite_psa_crypto.function

```diff
@@ -31,6 +31,29 @@
 #define ASSERT_OPERATION_IS_ACTIVE(   operation ) TEST_ASSERT( operation.id != 0 )
 #define ASSERT_OPERATION_IS_INACTIVE( operation ) TEST_ASSERT( operation.id == 0 )
 
+#if defined(PSA_WANT_ALG_JPAKE)
+void ecjpake_operation_setup( psa_pake_operation_t *operation,
```

Comment:
This function can cause a test failure, so it should return a success/failure status to its caller, and its caller should `goto exit` on failure.

We won't miss test failures with the current code, and the test framework guarantees that it reports the first test failures, but continuing the test after a failure adds confusion when debugging.